src/client/components/EventDepartments.js

```diff
@@ -34,11 +35,87 @@ const LeadInfo = lead => (
   </div>
 );
 
-const DepartmentNumbers = () => (
-  <div className={concat(style.numberSection, theme.divider)} style={{ minHeight: '200px' }}>
-    TODO
-  </div>
-);
+const DepartmentNumbers = ({ departments }) => {
+
+  // This is fucking awful, Michael. Definitely do it better when there's a real data model.
```

Comment:
...but...but there is a real data model, Michael.

Reply:
I'm not sure it's correct to have the UI data model be 1:1 with the database structure. It might be, but I wanna think about it more.

Reply:
It's probably not correct. I didn't realize you were talking about another data model (the UI one) and not the persistent data model.